### PR TITLE
/noinfer is default behaviour

### PR DESCRIPTION
**Pull request by rospoly**
targeting #204

### CODE REVIEW

File: Source/Core/CommandLineOptions.cs [hunk 1]
```diff
@@ -1822,7 +1822,7 @@ checking errors
                    s = debug statistics
                 0..9 = number of iterations before applying a widen (default=0)
   /noinfer      turn off the default inference, and overrides the /infer
-                switch on its left
+                switch on its left (default)
```
Comment (zvonimir) [inline Source/Core/CommandLineOptions.cs:1825]: Wait, why do we need `/noinfer` if the default behavior now is to not infer?
It seems to me that we do not need this switch any more.

Comment (zvonimir) [inline Source/Core/CommandLineOptions.cs:1825]: Also, above it says `The default is /infer:i`. That is not true any more, and it should be removed.
You can say something like `If this flag is omitted, Boogie does not attempt to infer invariants.`

**Comment by zvonimir** [comment]
It seems that there are regressions now failing because I would guess they relied on `/infer:i` being the default behavior. This should be fixed by adding `/infer:i` when invoking Boogie on those regressions.

**Comment by shazqadeer** [comment]
Issue #31 is related to this PR.  Please take a look.

**Comment by bkragl** [comment]
Hey @zvonimir & @rospoly, I already implemented this at some point locally and fixed the regressions. I'll create another PR and close this one. Hope that's fine for you.

**Comment by zvonimir** [comment]
That's fine. Let me close this one then.

**Pull request closed without merging**